src/OidcClientSettings.ts

```diff
@@ -42,14 +42,24 @@ export interface OidcClientSettings {
     redirect_uri: string;
     /** The OIDC/OAuth2 post-logout redirect URI */
     post_logout_redirect_uri?: string;
+
+    /** Indicates when sending client secret if sent as a post param or in `Authorization` header using HTTP Basic (default: `client_secret_post`) */
```

Comment:
This doc string could need better wording.

Reply:
yes, that was really not good, i have changed that now

Reply:
yes, that was really not good, i have changed that now